python/python/lancedb/table.py

```diff
@@ -1424,6 +1424,7 @@ def create(
         on_bad_vectors: str = "error",
         fill_value: float = 0.0,
         embedding_functions: List[EmbeddingFunctionConfig] = None,
+        storage_options=None,
```

Comment:
Is this needed?  I though the sync API didn't have storage_options?

Reply:
Ah sorry it shouldn't be in here. Thanks.